### ISSUE TITLE
Add 16 KB support for Android 15

### ISSUE THREAD

**Issue by smore-lore** [google/glog#1115]
Hi, Android 15 is introducing a 16 KB page size development mode and asking new apps to work in this mode, so that future devices can use 16 KB pages. You may want to ensure your SDK works in this mode, so that app developers using this software package don't have to update themselves. Thank you!

Read more at https://developer.android.com/16kb-page-size.

**Comment by sergiud** [comment]
Thanks for the heads up.

Do you see any specific issues that need to be fixed in glog? Other than that, we do not provide Android binaries. Users can decide themselves whether they enable 16 kb page size support or not.

**Comment by smore-lore** [comment]
No, I don't see any specific issues. 16 KB page size should be the default build alignment - it will work on 4 KB and 16 KB devices.
